src/task.c

```diff
@@ -53,24 +53,15 @@ static inline void sanitizer_finish_switch_fiber(void) {}
 #endif
 
 #if defined(JL_TSAN_ENABLED)
-static inline void tsan_destroy_ctx(jl_ptls_t ptls, jl_ucontext_t *ctx) {
-    if (ctx != &ptls->root_task->ctx) {
-        __tsan_destroy_fiber(ctx->tsan_state);
+static inline void tsan_destroy_ctx(jl_ptls_t ptls, void *state) {
+    if (state != &ptls->root_task->state) {
+        __tsan_destroy_fiber(ctx->state);
```

Comment:
This line, and line 60, refer to `ctx`, but that variable is no longer defined, so this cannot possibly compile. What's supposed to happen here?